pick.c

```diff
@@ -371,10 +379,16 @@ selected_choice(void)
 			 * move the cursor if the position is non zero.
 			 */
 			tty_putp(tty_parm1(parm_right_cursor, j), 1);
-		tty_putp(cursor_normal, 0);
+		if (vi_mode)
+			tty_putp(cursor_visible, 0);
```

Comment:
Although this will not work on all terminals, it seems the only "portable" way to change the cursor appearance (as an indicator that the Vi command-mode is active) ... the alternative is: https://vt100.net/docs/vt510-rm/DECSCUSR.html